code/modules/mob/living/carbon/human/species_types/podpeople.dm

```diff
@@ -77,5 +77,32 @@
 		return TRUE
 	return ..()
 
+/datum/species/pod/create_pref_unique_perks()
+	var/list/to_add = list()
+
+	to_add += list(list(
+		SPECIES_PERK_TYPE = SPECIES_NEUTRAL_PERK,
+		SPECIES_PERK_ICON = "lightbulb",
+		SPECIES_PERK_NAME = "Photosynthetic",
+		SPECIES_PERK_DESC = "As long as you are concious, and within a well-lit area, you will slowly brute, burn, toxin and oxygen damage and gain nutrition - and never get fat! \
+		However, if you are LOW on nutrition, you will progressively take brute damage until you die or enter the light once more."
+	))
+
+	to_add += list(list(
+		SPECIES_PERK_TYPE = SPECIES_NEGATIVE_PERK,
+		SPECIES_PERK_ICON = "biohazard",
+		SPECIES_PERK_NAME = "Weedkiller susceptability",
+		SPECIES_PERK_DESC = "Being a floral life form, you are susceptable to anti-florals and will take extra toxin damage from it!"
+	))
+
+	to_add += list(list(
+		SPECIES_PERK_TYPE = SPECIES_NEGATIVE_PERK,
+		SPECIES_PERK_ICON = "briefcase-medical",
+		SPECIES_PERK_NAME = "Semi-Complex Biology",
+		SPECIES_PERK_DESC = "Your biology is extremely complex, and ordinary health analyzers will be unable to scan you!"
```

Comment:
```suggestion
		SPECIES_PERK_DESC = "Your biology is extremely complex, and ordinary health analyzers will be unable to scan you! Try a plant analyzer instead."
```

Reply:
Good idea, i'll also include that advanced analayzers can be used (later)